### DISCUSSION TITLE
Photopea becomes unresponsive after a bit

### DISCUSSION THREAD

**Discussion by OhSeanO** [photopea/photopea#6779] [General]
Hello. First off, I want the Devs to know I love this tool and use it all the time and tell other people to use it. I have even donated to it.

My issue is that after I've had it open for awhile I consistently slows down a lot and then crashes and I lose all my work. I obviously have learned to save often, but sometimes when I'm zoning out making a card game or something I forget. I'm not usually the type to post something, usually I just lurk in forums, but there are no open posts I could find about this issue. This happens on multiple devices, including this macbook, my parents imac, my lenova legion and every school computer I have used. It seems like it uses an inordinate amount of memory and eventually can't handle it and crashes. 

I did all the things it says to do on other topics that have been closed on this issue, such as allowing for more memory in photopea, disabling all extensions and limiting the number of tabs I have open. I keep chrome up to date. Please help. I'd much rather use this than bite the bullet and pay for adobe.

**Comment by photopea** [comment]
Is there any chance that I could see it unresponsive on my computer? What should I do? I dont usually use Photopea for more than 5 minutes. Also, are ads running or you are blocking them? How much RAM memory do you have?
